app/main/menus.js

```diff
@@ -7,6 +7,11 @@ import i18n from '../configs/i18next.config';
 let menuTemplates = {mac: {}, other: {}};
 let mainWindow = null;
 
+const appVersionTitle = i18n.t('Appium Desktop');
+const appVersionMessage = `App Version: ${app.getVersion()}\n` +
```

Comment:
`App Version` should also be translated.
I would recommend putting the whole string into resources though, because in RTL languages it might be spelled differently even if the particular words don't need to be translated

Reply:
ah, RTL... Forgot it